script.js

```diff
@@ -148,10 +200,53 @@ function currencyDisplayTable(event) {
         //creating cells
         row.insertCell(0).textContent = currency.base;
         row.insertCell(1).textContent = quoteCurrency;
-        row.insertCell(2).textContent = currency.rates[quoteCurrency];
+        if (
+          specialRates[baseCurrency] &&
+          specialRates[baseCurrency][quoteCurrency]
+        ) {
+          const specialRate = specialRates[baseCurrency][quoteCurrency];
+          row.insertCell(2).textContent = `${currency.rates[quoteCurrency]} ${
+            currentRate >= specialRate
+              ? "Oh  :fire:  Don't trade today.The conversion rate is too high"
```

Comment:
Since 🔥 its part of ASCII you must insert it here as an emoji to be display, meaning either you can copy the emoji via CTRL + C from here 🔥 and paste it to the code via CTRL + V or if you use mac you can press Control + CMD + Space to display emoji table where you can pick the emoji :)

Reply:
Thank you, done.